src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.css';
+
+const list = [
+  {
+    description: 'Grocery Shopping',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Finish Homeworks',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Pay Bills',
+    completed: false,
+    index: 3,
+  },
+];
+const generatedList = document.querySelector('.generated-list');
+
+const displayTodo = () => {
+  const displayTodo = list.map((item) => `
```

Comment:
- [ ] Kindly sort the list before displaying it as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.). The list should appear in order of the index values for each task.